### DISCUSSION TITLE
Build tasmota-tls at release

### DISCUSSION THREAD

**Discussion by jeroenst** [arendst/Tasmota#17407] [Q&A] [answered]
Is it possible to add a tasmota-tls and tasmota32-tls with tls enabled to the build of released software by default?

I have an automatic build on my own server for this but it would be nice if this would be included in future releases.

**Comment by sfromis** [comment, marked as answer]
You need no `tasmota32-tls` as `USE_MQTT_TLS` is *already* included in `tasmota32` builds.

And you do have an "unofficial" TLS dev build in https://github.com/tasmota/install/tree/main/firmware/unofficial

Also, you do have a release ESP8266 build including the TLS feature, `tasmota-zbbridge`

**Reply by jeroenst** [reply]
Perfect!

**Reply by jeroenst** [reply]
Thanx!
